puresasl/mechanisms.py

```diff
@@ -13,6 +14,11 @@
 except ImportError:
     have_kerberos = False
 
+if platform.system() == 'Windows' :
+    import winkerberos as kerberos
```

Comment:
Can you wrap this in a `try/except`, like the kerberos import just above this?

Reply:
Yep, thought of that this morning, ill make that change